components/service/glean/README.md

```diff
@@ -109,6 +109,16 @@ object parameter of the `Glean.initialize` method. For example:
 Glean.initialize(applicationContext, Configuration(channel = "beta"))
 ```
 
+### Enabling and disabling metrics
+
+`Glean.setUploadEnabled()` should be called in response to the user enabling or
+disabling telemetry.  This method should also be called at least once prior
+to calling `Glean.initialize()`.
```

Comment:
Post-Fenix-MVP thought:
Can we have a follow-up bug on considering to make this setting sticky, so the "should be called at least once prior" requirement falls away?
I don't remember exactly where our discussions went on this, but from the embedding products side, this seems like a footgun (or "just one more thing to remember") that we might be able to hide away.

Reply:
Sure.  I filed [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1555328)